R/minimize.R

```diff
@@ -17,12 +17,12 @@
 #' @export
 minimize <- function(objective, subject_to, initial_design,
                      lower_boundary_design, upper_boundary_design,
-                     c2_monotone = F,
-                     post_process = F,
+                     c2_monotone = FALSE,
+                     post_process = FALSE,
                      opts = list(
                          algorithm   = "NLOPT_LN_COBYLA",
-                         xtol_rel    = 1e-4,
-                         maxeval     = 2500
+                         xtol_rel    = 1e-5,
+                         maxeval     = (200 * length(initial_design@c2_pivots)^2)
```

Comment:
this could get out of hands quickly... any rationale for this particular formula? I'd just go with a fixed default value based on whatever choice of the number of pivots we use as default.

Reply:
alright, I'll add an todo and use 5000 in the meanwhile

Reply:
hm, rather 10 or 20k, right?